ebean-core/src/test/java/org/tests/idkeys/db/GenKeySequence.java

```diff
@@ -4,12 +4,14 @@
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
+import javax.persistence.SequenceGenerator;
 
 @Entity
 public class GenKeySequence {
   public final static String SEQUENCE_NAME = "SEQ";
 
   @Id
+  @SequenceGenerator(name = "SEQ_NAME", sequenceName = GenKeySequence.SEQUENCE_NAME)
```

Comment:
Hmmm, I think this should be removed / we should not add `@SequenceGenerator` here.

Reply:
In Ebean 11 `@SequenceGenerator(name = "SEQ_NAME", sequenceName = GenKeySequence.SEQUENCE_NAME)` is annotated on the Entity itself. This is no longer possible.

* rith this annotation the sequence is named `SEQ` in the migration scripts.
* without this annotation the sequence is named `SEQ_NAME` in the migration script (and such the test fails currently).
* renaming the static variable `SEQUENCE_NAME = "SEQ_NAME"` may also fix this test.

I decided to reuse the solution of Ebean 11 by annotating the field, because I thought this was done on purpose ;)

Should I remove the annotation and rename `SEQUENCE_NAME = "SEQ_NAME"` ?

Reply:
It isn't clear why this expects and needs the additional `@SequenceGenerator` when `@GeneratedValue` has those attributes.  So we need to understand why that is / is there a behaviour change here etc

Reply:
We removed this change from this pull request, as it does not have anything to do with sequences in MariaDB (except that we wanted to test sequences).

We provided a different pull request to address this situation:
https://github.com/ebean-orm/ebean/pull/2308